statsd/statsd.go

```diff
@@ -106,7 +106,7 @@ type Client struct {
 // "unix:///path/to/socket".
 func New(addr string) (*Client, error) {
 	if strings.HasPrefix(addr, UnixAddressPrefix) {
-		w, err := newUdsWriter(addr[len(UnixAddressPrefix)-1:])
+		w, err := newAsyncUdsWriter(addr[len(UnixAddressPrefix)-1:])
```

Comment:
What if the user wants the blocking writer? Also, please see my note below bout using the async writer as a blocking writer by changing the channel depth.

Reply:
Dealing with this in #82